galsim/interpolatedimage.py

```diff
@@ -525,7 +526,7 @@ def buildNoisePadImage(self, noise_pad_size, noise_pad, rng):
                 # Make sure that we are using a specified RNG by resetting that in this cached
                 # CorrelatedNoise instance, otherwise preserve the cached RNG
                 noise = noise.copy(rng=rng)
-        elif isinstance(noise_pad, str):
+        elif isinstance(noise_pad, basestring):
```

Comment:
This isn't covered, but adding this coverage isn't something I want to tackle right now.  (Several other branches here also need coverage.)  I think it's fine to leave this one line uncovered for this PR, unless someone feels inspired to add some more variants to `test_corr_padding` in test_interpolatedimage.py.